src/System.Runtime.Serialization/src/project.json

```diff
@@ -1,7 +1,6 @@
 {
   "dependencies": {
-    "NETStandard.Library": "1.6.0-rc4-24214-03",
-    "Microsoft.Win32.Registry": "4.0.0-rc4-24214-03",
+    "NETStandard.Library": "1.6.0-rc4-24214-03"
```

Comment:
Can you also add some of the serialization packages here? That should get us more types for free as they are already implemented. I would expect to see your compat baseline be reduced once you add those.